CONTRIBUTING.md

```diff
@@ -114,13 +114,11 @@ class ExampleClass{
 ```
 
 <!-- TODO: RFC and voting on the forums instead -->
-### RFC and Voting
-* These are big Pull Requests or contributions that change important behavior.
-* RFCs will be tagged with the *Type: Request For Comments* label
-* A vote will be held once the RFC is ready. All users can vote commenting on the Pull Request
-* Comments MUST use "Yes" or "No" on the FIRST sentence to signify the vote, except when they don't want it to be counted.
+### Major API changes
+* Major API changes, no matter API-breaking or not, SHOULD be proposed by creating a topic in [Discussions](https://github.com/pmmp/PocketMine-MP/discussions).
+* The change proposal enters the "Voting Period" of 7 days after the team determines that sufficient discussion has been made.
```

Comment:
is 7 days really long enough? Most popular change proposals and enhancements don't even get 20 votes after being open for a month.

Reply:
Or do we just dismiss the voting period and just make 7 days the "final comment period"?

Reply:
I don't think 7 days is in any way practical.

Reply:
What I mean is that there shouldn't be votes at all, and the last 7 days are just a way to notify people that the RFC is finalizing soon. They could always have commented in the months before the finalization period, and 7 days is just a reminder.
Otherwise, just don't have a reminder and finalize the RFC straight away when the time comes.